adapter/api/src/main/java/org/apache/rocketmq/eventbridge/adapter/api/converter/HttpEventConverter.java

```diff
@@ -0,0 +1,293 @@
+import org.apache.rocketmq.eventbridge.domain.cache.EventSourceCacheService;
+import org.apache.rocketmq.eventbridge.domain.model.source.EventSource;
+import org.apache.rocketmq.eventbridge.exception.EventBridgeException;
+import org.apache.rocketmq.eventbridge.tools.ARNHelper;
+import org.apache.rocketmq.eventbridge.tools.NetUtil;
+import org.apache.rocketmq.eventbridge.tools.transform.Data;
+import org.apache.rocketmq.eventbridge.tools.transform.StringData;
+import org.apache.rocketmq.eventbridge.tools.transform.Transform;
+import org.apache.rocketmq.eventbridge.tools.transform.TransformBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.server.reactive.ServerHttpRequest;
+import org.springframework.stereotype.Service;
+import org.springframework.util.CollectionUtils;
+
+import java.lang.reflect.Type;
+import java.net.URI;
+import java.nio.charset.StandardCharsets;
+import java.time.OffsetDateTime;
+import java.time.ZonedDateTime;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+
+import static org.apache.rocketmq.eventbridge.domain.common.exception.EventBridgeErrorCode.JSON_ATTRIBUTE_INVALID;
+import static org.apache.rocketmq.eventbridge.domain.common.exception.EventBridgeErrorCode.PutEventsRequestSecurityCheckFailed;
+import static org.apache.rocketmq.eventbridge.domain.model.source.HTTPEventSourceService.SECURITY_CONFIG_IP;
+import static org.apache.rocketmq.eventbridge.domain.model.source.HTTPEventSourceService.SECURITY_CONFIG_NONE;
+import static org.apache.rocketmq.eventbridge.domain.model.source.HTTPEventSourceService.SECURITY_CONFIG_REFERER;
+
+/**
+ * @Author changfeng
+ * @Date 2022/4/25 11:28 上午
+ */
+@Service
+public class HttpEventConverter {
+    private static final Logger logger = LoggerFactory.getLogger(HttpEventConverter.class);
+
+    @Autowired
+    EventSourceCacheService eventSourceCacheService;
+
+    private static final String TYPE = "eventbridge:Events:HTTPEvent";
+    private static final String DATA_CONTENT_TYPE = "application/json";
+
+    private static final String HEADER_X_REAL_IP = "x-real-ip";
+    public static final String HEADER_EVENTBRIDGE_DATE = "x-eventbridge-date";
+
+    private static final String SECURITY_CONFIG = "SecurityConfig";
+    private static final String IP_CONFIG = "Ip";
+    private static final String METHOD_CONFIG = "Method";
+    private static final String REFERER_CONFIG = "Referer";
+
+    public static final String EXTENSION_REGIONID = "aliyunregionid";
+    public static final String EXTENSION_ACCOUNTID = "aliyunaccountid";
+    public static final String EXTENSION_PUBLISHADDR = "aliyunpublishaddr";
```

Comment:
Remove the code around aliyun, if needed , change to the SPI  and move the impl to rpc package.

Reply:
The use of these configurations has been moved to the impl of rpc package